templates/database_cleaner_rspec.rb

```diff
@@ -1,21 +1,28 @@
 RSpec.configure do |config|
+  config.use_transactional_fixtures = false
```

Comment:
I like the rename to `database`, as that means more db configuration can be moved out of the pure spec-helper. Which is nice.